scapy/modules/nmap.py

```diff
@@ -196,15 +203,17 @@ def nmap_search(sigs):
 
 @conf.commands.register
 def nmap_fp(target, oport=80, cport=81):
+    # type: (str, int, int) -> Tuple[Union[int, float], List]
     """nmap fingerprinting
-nmap_fp(target, [oport=80,] [cport=81,]) -> list of best guesses with accuracy
-"""
+    nmap_fp(target, [oport=80,] [cport=81,]) -> list of best guesses with accuracy
+    """
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok